cmd/kubectl-gadget/utils/trace.go

```diff
@@ -1037,7 +1037,12 @@ func genericStreams(
 	for {
 		select {
 		case msg := <-completion:
-			fmt.Fprintln(os.Stderr, msg)
+			output := os.Stdout
+			if strings.HasPrefix(msg, "Error") {
+				output = os.Stderr
+			}
+			fmt.Fprintln(output, msg)
+
```

Comment:
It seems to hacky to me. I'd rather remove the "completed" message than having this logic here.